ROMFS/px4fmu_common/init.d/CMakeLists.txt

```diff
@@ -31,6 +31,8 @@
 #
 ############################################################################
 
+add_subdirectory(airframes)
```

Comment:
How did this work before?

Reply:
Looks like `px_process_airframes.py` does not use the airframes listed in this directory's CMakelists.txt.

```
COMMAND ${PYTHON_EXECUTABLE} ${PX4_SOURCE_DIR}/Tools/px_process_airframes.py
		--airframes-path ${romfs_gen_root_dir}/init.d
		--start-script ${romfs_gen_root_dir}/init.d/rc.autostart
		--board ${PX4_BOARD}
```

Reply:
I'm still confused in how this used to work and how it's supposed to work. Is it clear to you?

Reply:
Three things are needed for it to work:
- the python script generates a shell script rc.autostart that dispatches airframe scripts at startup. The python script lists files in the directory, so it does not need the cmake airframe list. That is why this part worked before the PR. 
- The airframe scripts should be copied into the ROMFS. I am not clear where this was done, considering that the airframe scripts were not listed by cmake before this PR. 
- The generation should be te-triggered when an airframe script is modified. This is done by this PR

Reply:
Eww, point 2 is done with this cmake directive:
```
COMMAND ${CMAKE_COMMAND} -E copy_directory ${romfs_src_dir} ${romfs_gen_root_dir}
```

The whole folder is copied into ROMFS, then some files are pruned... Kinda would need a cleanup.

Reply:
@julianoes is that clearer?

Reply:
Yes thanks. So it needs more work to be clean?

Reply:
No, this PR is done.

Cleaning is non trivial: the cmake and python scripts should be modified to copy and parse only the airframes listed in the CMakelists.

Reply:
Ok I get it now.

I guess I'm referring to this when I say "more work is needed":
https://github.com/PX4/Firmware/blob/e23e3d7baed6871841911a3aa19812ddb1547b6b/ROMFS/CMakeLists.txt#L89-L90

Reply:
Yes indeed

Should we open an issue to track this TODO?

Reply:
I started but it's indeed a bit more complicated.